portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,35 +26,45 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private ArrayList<String> messages;
+  private Comments c;
 
   @Override
   public void init() {
     messages = new ArrayList<>();
```

Comment:
I think `messages` is unused now so it would make sense to delete the variable too.